npm/package.json

```diff
@@ -17,7 +17,7 @@
       "node": "./dist/main.js",
       "default": "./dist/browser.js"
     },
-    "./worker": "./dist/worker-browser.js"
+    "./worker": "./dist/compiler/worker-browser.js"
```

Comment:
I haven't yet exposed an entrypoint for the language service, because there are no external consumers. That will probably follow in a later PR when I wire up VS Code and playground.

Reply:
The later PR: #429